miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/analytics/MiniAppAnalytics.kt

```diff
@@ -17,17 +17,17 @@ private inline fun <T> whenHasAnalytics(callback: () -> T) {
 
 /** Only init when analytics dependency is provided. */
 @Suppress("SwallowedException", "TooGenericExceptionCaught")
-internal class MiniAppAnalytics(val rasProjectId: String) {
+class MiniAppAnalytics(private val rasProjectId: String) {
```

Comment:
I'm not sure that you need to make this class public - usually we use `MiniApp` as our main public interface. You could leave `MiniAppAnalytics` as internal, then add new public functions in  `MiniApp.addAnalyticsConfig` and `MiniApp.removeAnalyticsConfig` which then just call `MiniAppAnalytics.instance().addAnalyticsConfig`.